app/src/layouts/common/projectSelector/projectSelector.jsx

```diff
@@ -148,17 +147,29 @@ export class ProjectSelector extends Component {
                 data-placement={placement}
               >
                 <ScrollWrapper autoHeight autoHeightMax={600}>
-                  {projects.map((project) => (
-                    <NavLink
-                      to={{ type: PROJECT_PAGE, payload: { projectId: project } }}
-                      key={project}
-                      className={cx('project-list-item')}
-                      activeClassName={cx('active')}
-                      onClick={this.onClickProjectName}
-                    >
-                      <span title={project}>{project}</span>
-                    </NavLink>
-                  ))}
+                  {Object.keys(projects)
+                    .sort()
+                    .map((projectKey) => {
+                      const { projectSlug: projectName, organizationSlug } = projects[projectKey];
+
+                      return (
+                        <NavLink
+                          to={{
+                            type: PROJECT_PAGE,
+                            payload: {
+                              projectKey,
+                              organizationSlug,
+                            },
+                          }}
+                          key={projectKey}
+                          className={cx('project-list-item')}
+                          activeClassName={cx('active')}
+                          onClick={this.onClickProjectName}
+                        >
+                          <span title={projectName}>{projectName}</span>
```

Comment:
I suppose we need to use project name rather than project slug.
Project slug is URL only feature.

Reply:
Done.